documentation/milestones/milestone_template.md

```diff
@@ -45,50 +88,37 @@ The following work will *not* be completed as part of this milestone:
 ## Integrations
 
 ### Translations
+<!-- Optional -->
 
 Does this milestone involve delivering any content that needs translation?
 
 If so, when will English-language content be locked? Then when will translation be
 started and completed?
 
 ### Services going into PROD for the first time
+<!-- Optional -->
 
 This can include services going into PROD behind a feature flag that is not turned on.
 
 1. [to be added]
 
 ### Services being integrated in PROD for the first time
+<!-- Optional -->
 
 Are there multiple services that are being connected for the first time in PROD?
 
 1. [to be added]
 
 ### Data being shared publicly for the first time
+<!-- Optional -->
 
 Are there any fields being shared publicly that have never been shared in PROD before?
 
 1. [to be added]
 
 ### Security considerations
+<!-- Optional -->
 
-## Delivery
+Are there any security vulnerabilities that need to be addressed as part of this milestone?
```

Comment:
I think this is not actually the question I meant, although it's relevant. With security, I'm mostly thinking about: does completing this milestone open up new vectors of security threats. E.g., we're integrating with the existing Grants DB, so that means we're now adding risks due to live production data being connected.